packages/ui/src/components/item-layout/item-layout.web.tsx

```diff
@@ -83,7 +80,7 @@ export function ItemLayout({
         </Stack>
         {showChevron && (
           <ChevronRightIcon
-            color={pressableChevronStyles}
+            className={pressableChevronStyles}
```

Comment:
Nice fix! 

Maybe we can just declare `pressableCaptionStyles` inline instead as it's only used once

Reply:
<img width="740" alt="image" src="https://github.com/user-attachments/assets/1cf57240-7ff2-4369-94f8-efaba5208eab" />
Not related to this PR, but also seems to be used in more than one place.